akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/ReplayingEvents.scala

```diff
@@ -180,7 +180,7 @@ private[akka] final class ReplayingEvents[C, E, S](
 
   protected def onRecoveryCompleted(state: ReplayingState[S]): Behavior[InternalProtocol] = try {
     tryReturnRecoveryPermit("replay completed successfully")
-    setup.recoveryCompleted(state.state)
+    setup.onSignal(akka.persistence.typed.RecoveryCompleted(state.state))
```

Comment:
top level import

Reply:
I had a name conflict with something from untyped, but I guess I can import `typed` instead or do an alias.

Reply:
I don't know where I got that from, no conflict, nvm